schema/spacewalk/postgres/tables/suseSaltEvent.sql

```diff
@@ -3,7 +3,8 @@
 CREATE TABLE suseSaltEvent (
   id SERIAL PRIMARY KEY,
   minion_id CHARACTER VARYING(256),
-  data TEXT NOT NULL
+  data TEXT NOT NULL,
+  queue SMALLINT NOT NULL
```

Comment:
I would recommend using the otherwise-standard `NUMERIC` type here.

Reply:
Done. Updated the upgrade schema too